code/ui/manager/src/components/mobile/navigation/MobileNavigation.tsx

```diff
@@ -0,0 +1,82 @@
+import type { FC } from 'react';
+import React, { useState } from 'react';
+import { styled } from '@storybook/theming';
+import { IconButton, Icons } from '@storybook/components';
+import { MobileMenuDrawer } from './MobileMenuDrawer';
+import { MobileAddonsDrawer } from './MobileAddonsDrawer';
+import Panel from '../../../container/Panel';
+
+interface MobileNavigationProps {
+  storyTitle?: string | null | undefined;
+}
+
+export const MobileNavigation: FC<MobileNavigationProps> = ({ storyTitle }) => {
+  const [isMenuOpen, setMenuOpen] = useState(false);
+  const [isAddonsOpen, setAddonsOpen] = useState(false);
+  const [isAboutOpen, setAboutOpen] = useState(false);
+
+  const closeMenu = () => {
+    setMenuOpen(false);
+
+    setTimeout(() => {
+      setAboutOpen(false);
+    }, 300);
```

Comment:
why does this need to be in a setTimeout? I think that can cause trouble.

Maybe we can use `onExited` on the drawer transition if it's to stagger the transitions.
https://reactcommunity.org/react-transition-group/transition#Transition-prop-onExited